### ISSUE TITLE
Issue compiling with sass ruby gem

### ISSUE THREAD

**Issue by riri** [jgthms/bulma#478]
### Overview of the problem

  * This is about the Bulma **CSS framework**
  * I'm using Bulma **version** 0.3.1
  * My **browser** is: Chromium
  * This is a **Sass** issue: I'm using sass ruby gem version 3.4.23
  * No result in issues search

### Description

I have imported bulma for using its sass modules, in my [Brunch](http://brunch.io) project, using the [sass library from ruby gem](https://rubygems.org/gems/sass/versions/3.4.23). I face a compilation error with the `bulma/utilities/mixins.sass` file:

```
Error: Invalid CSS after "1000px ": expected expression (e.g. 1px, bold), was "!default // 960..."
        on line 176 of sass/utilities/mixins.sass
        from line 5 of sass/utilities/_all.sass
        from line 4 of bulma.sass
  Use --trace for backtrace.
```

The problem seems to be the ending comments.

### Steps to Reproduce

1. If needed, install sass (sic):
```
gem install sass
```

2. From the bulma directory
```
sass -c bulma.sass
```

### Normal or not?

I don't know if this behavior is the standard one or not, maybe other processors (like `node-sass`) are more flexibles. I have remarked my syntax highlighter in Atom does not render the comments following `!default` either, so...

Maybe it's just about moving comments above their respective lines?

**Comment by yoostk** [comment]
I think this issue was fixed in #446.

I cloned the bulma's latest master branch and compiled using Sass gem (ver3.4.23) without a problem. `$ sass -c bulma.sass` prints nothing.

**Comment by riri** [comment]
I will have to refine my searches in the future :)

Thanks

**Comment by melloc01** [comment]
Confirmed when downloading the .zip file from the website.  **v 0.3.1**
This issue does not show if you download the .zip from _Github_. **master**

Think there should there be a **v 0.3.2** that fixes this issue asap?

**Comment by yoostk** [comment]
@melloc01 I guess so. See https://github.com/jgthms/bulma/issues/505#issuecomment-277515707.

**Comment by melloc01** [comment]
@yoostk, yep! =) thanks

**Comment by guillecro** [comment]
I am having the same problem using **compass watch**, by importing **bulma.scss**

` error web/assets/sass/es-formulario.scss (Line 176 of bower_components/bulma/sass/utilities/mixins.sass: Invalid CSS after "1000px ": expected expression (e.g. 1px, bold), was "!default // 960...")`

**Comment by razlanisme** [comment]
Same issue encountered @guillecro

**Comment by guillecro** [comment]
+Kudos for @melloc01
Issue doesn't show if you download the .zip from the master branch.

**Comment by jgthms** [comment]
Fixed in 0.3.2
